Source/WebKit/Shared/RemoteLayerTree/RemoteLayerBackingStore.mm

```diff
@@ -60,6 +60,32 @@
 
 using namespace WebCore;
 
+constexpr Seconds delegatedContentsFinishedTimeout = 5_s;
```

Comment:
This value is now cut-n-pasted into 3 locations. Does it require a note that if it ever changes to update the other values?